testDBConnection.php

```diff
@@ -22,6 +22,12 @@
             echo $test->get();
         ?>
     </p>
+    <h2>Attempt</h2>
+    <p>
+        <?php
+            $test->getConnectDB();
```

Comment:
added this part to see what was actually getting returned when using `connect_db()`